scripts/install/conda-packages-list.txt

```diff
@@ -1,3 +1,33 @@
-pandas
+astroplan
+astropy >= 3.0.0
+ccdproc
+codecov
+coveralls
 cython
+dateparser
+google-cloud-sdk
+google-cloud-storage
+matplotlib >= 2.0.0
+numpy >= 1.12.1
+pycodestyle == 2.3.1
+pymongo >= 3.2.2
+pyserial >= 3.1.1
+pytest >= 3.4.0
+pandas
+python
```

Comment:
Does this actually install the python version from here and if so should we specify a version?

Reply:
Specifying a version is problematic. If you say ==3.7, then the constraints are unsolvable.

Reply:
Gotcha.